eclair-core/src/main/resources/reference.conf

```
@@ -17,7 +17,7 @@ eclair {
   }
 
   // override this with a script/exe that will be called everytime a new database backup has been created
-  backup-notify-script = ""
+  # backup-notify-script = ""
```

Could we provide an example instead of an empty string (which would not work anyway)?

```suggestion
  # backup-notify-script = "/path/to/script.sh"
```